src/Modules/OrchardCore.Commerce/Assets/Styles/shopping-cart-widget.scss

```diff
@@ -0,0 +1,19 @@
+.widget-shopping-cart-widget {
+    position: absolute;
+    right: 10px;
+    top: 10px;
```

Comment:
Please don't repeat related literals. Create CSS variables, e.g. `--shopping-cart-widget-offset` and `--shopping-cart-widget-circle-size`.